python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py

```diff
@@ -94,9 +94,9 @@ def optimize_llm_pre(model: torch.nn.Module, qtype, mixed_precision):
                 not cpu_lm_head:
             # Do not split lm_head and use sym_int8 instead when mixed_precison is True
             is_split = (not mixed_precision) and qtype == "sym_int4_rtn"
-            split_num = 14 if is_split else 1
+            split_num = 28 if is_split else 1
```

Comment:
I'm considering whether we should expose the use_split option to users.
###  Y (make split_num=14 if use_split=False)
- Pros: We can recommend the users set `use_split=False` to avoid the performance degradation on driver versions below 2715
- Cons: It may need to be clarified for the user.

### N (should update the readme recommended driver section)
- Pros: This approach is more straightforward for users, and less configuration is needed.
- Cons: Users must ensure they use driver version 2908 or higher. For MTL users, we would need to fall back to split_num=14 w/ slice.

Please let me know if you have any comments. @Oscilloscope98 @jason-dai